CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/templates/SimpleEditPlugin.java

```diff
@@ -106,7 +106,7 @@ public final void run(final PluginGraphs graphs, final PluginInteraction interac
 
                     try {
                         description = describedEdit(writableGraph, interaction, parameters);
-                    } catch (final Exception ex) {
+                    } catch (final PluginException | InterruptedException ex) {
```

Comment:
can you split out the interrupted exception and add a thread interrupt to that one?

Reply:
Would something like this be suitable?
`if(ex instanceof InterruptedException){
        Thread.currentThread().interrupt();
}`

Reply:
Nevermind, I've split it into two catch statements